### PR TITLE
Apid Fixes

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Allows Apidite to be spoken and fixes the sprites for Apid Wings.

## Why It's Good For The Game

It'd be nice to allow a language to be spoken and to see an object.

## Changelog
:cl:
fix: Fixed Apidite, supposedly.
fix: Fixed apid wing sprites.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
God damn it, hold on.

File: code/modules/surgery/organs/tongue.dm [hunk 1]
```diff
@@ -18,7 +18,8 @@
 		/datum/language/ratvar,
 		/datum/language/aphasia,
 		/datum/language/piratespeak,
-		/datum/language/rlyehian,
+		/datum/language/rlyehian
+		/datum/language/apidite,
```
Comment (Crossedfall) [inline code/modules/surgery/organs/tongue.dm]:
```suggestion
		/datum/language/rlyehian,
		/datum/language/apidite,
```

Comment (ghost) [reply code/modules/surgery/organs/tongue.dm]: Yeah, I read through the Travis log and was fixing that.

**Comment by Crossedfall** [comment]
Well, the wings are back.. But 
wings

**Comment by ghost** [comment]
> Well, the wings are back.. But
> wings

To be frank I don't see the problem. 🤔

**Review by Crossedfall — APPROVED** [review]
Looks good. The language seems to be working now as well.

**Pull request merged**